godaddy-email-marketing.php

```diff
@@ -151,6 +158,7 @@ public function init() {
 
 		// Initialize settings.
 		if ( is_admin() ) {
+			$this->display_gem_notice = get_option( 'wpem_gem_notice' );
```

Comment:
@fjarrett You're right. This was originally placed in a private variable because it was used in more than one location (DRY). Without the AJAX this is un-necessary. Removing it now.